src/encoded/schemas/file.json

```diff
@@ -671,40 +680,41 @@
         },
         "supersedes": {
             "title": "Supersedes" ,
-            "description": "The files that this file replaces.",
+            "description": "The files that this file supersedes.",
+            "comment": "The files in this list should be obsolete and the superseeding file object should be used instead.",
             "type": "array",
             "uniqueItems": true,
             "items": {
-                    "comment": "See file.json for a list of available identifiers.",
+                    "comment": "The files in this list should be archived or revoked.",
                     "type": "string",
                     "linkTo": "File"
             }
         },
         "paired_with": {
             "title": "File pairing",
-            "description": "The file that corresponds with this file.",
-            "comment": "See file.json for a list of available identifiers.",
+            "description": "The paired end fastq that corresponds with this file.",
+            "comment": "This is required for files with paired_end 2 and is calculated for files with paired_end 1.",
             "type": "string",
             "linkTo": "File"
         },
         "dataset": {
             "title": "Dataset",
             "description": "The experiment or dataset the file belongs to.",
-            "comment": "For experiments, see experiment.json for available identifiers. For datasets, see dataset.json for available identifiers.",
+            "comment": "Required. See dataset.json for available identifiers.",
             "type": "string",
             "linkTo": ["Experiment", "Dataset", "Reference", "Annotation", "PublicationData", "UcscBrowserComposite"]
         },
         "replicate": {
             "title": "Replicate",
             "description": "The experimental replicate designation for the file.",
-            "comment": "See replicate.json for available identifiers.",
+            "comment": "This field is required for sra and fastq files. See replicate.json for available identifiers.",
             "type": "string",
             "linkTo": "Replicate"
         },
         "assembly": {
```

Comment:
I am not suggesting we change this not, but this property seems really out of place.   A property of analysis that could be denormalized (calculated) on to file.